translations/mod-fqm-manager/en.json

```diff
@@ -388,5 +388,28 @@
   "entityType.drv_organization_details.agreements.notes": "Notes",
   "entityType.drv_organization_details.agreements.notes._qualified": "Agreement notes",
   "entityType.drv_organization_details.acqunit_ids": "Acquisition unit ID list",
-  "entityType.drv_organization_details.acquisition_unit": "Acquisition unit list"
+  "entityType.drv_organization_details.acquisition_unit": "Acquisition unit list",
+  "entityType.drv_organization_details.accounts": "Organization account",
+  "entityType.drv_organization_details.accounts.name": "Account name",
```

Comment:
The non-qualified can just be `Name`, `Notes`, etc.

Reply:
Done deal @ncovercash